pkgs/misc/emulators/wine/sources.nix

```diff
@@ -12,6 +12,31 @@ let fetchurl = args@{url, sha256, ...}:
   pkgs.fetchFromGitHub { inherit owner repo rev sha256; } // args;
 in rec {
 
+  series3 = fetchurl rec {
```

Comment:
Are you not going to add it to pkgs/top-level/wine-packages.nix?

Reply:
Also, we could really use a comment with a list of upstream bugs that prevent wine 5 from working in your case.